tremor-connectors/tests/http/server.rs

```diff
@@ -13,14 +13,16 @@
 // limitations under the License.
 use anyhow::{anyhow, Result};
 use http::StatusCode;
-use hyper::body::HttpBody;
-use hyper::{body::to_bytes, Body, Request, Response};
+use http_body_util::BodyExt;
+use hyper::{body::Incoming, Request, Response};
 use hyper_rustls::HttpsConnectorBuilder;
+use hyper_util::{client::legacy::Client, rt::TokioExecutor};
 use std::{
     path::PathBuf,
     time::{Duration, Instant},
 };
 use tokio::time::timeout;
+use tremor_connectors::impls::http::utils::{empty_body, Body};
```

Comment:
both body and emptyu body are super small, one just being a type alias. I think it's worth to copy in the code for a single use here for not poluting the public interface of the connectors crate.

Reply:
Ja. That was scaffolding while I was whacking `struct Body` - next commit ...